apps/contact/views.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import messages
+from django.core.mail import EmailMessage
+from django.shortcuts import redirect, render_to_response
+from django.template import RequestContext
+
+from apps.contact.forms import ContactForm
+
+
+# Index page
+def index(request):
+    context = {'form': ContactForm}
+    return render_to_response('contact/index.html', context, context_instance=RequestContext(request))
+
+
+def contact_submit(request):
+    if request.method == 'POST':
+        form = ContactForm(request.POST)
+        if form.is_valid():
+            subject = '[Kontakt]' + form.cleaned_data['contact_name'] + 'har kontaktet dere gjennom ow4'
```

Comment:
```
>>> data['contact_name'] = "Sklirg"
>>> subject = '[Kontakt]' + data['contact_name'] + 'har kontaktet dere gjennom ow4'
>>> print(subject)
[Kontakt]Sklirghar kontaktet dere gjennom ow4
```

I still think you want spacing here.

As I suggested previously, use string substitution, like this:
```
>>> subject = '[Kontakt] {name} har kontaktet dere gjennom ow4'.format(name=data['contact_name'])
>>> print(subject)
[Kontakt] Sklirg har kontaktet dere gjennom ow4
```

However -- since name is not required, it might be more useful for the subject to be "Ny kontaktførespørsel fra online.ntnu.no" (note: We try to discourage the use of "ow4", since not everyone know what "ow4" is. We prefer "onlineweb" or "online.ntnu.no").